app/experimenter/nimbus-ui/src/components/Summary/TableAudience/index.tsx

```diff
@@ -76,6 +82,17 @@ const TableAudience = ({ experiment }: TableAudienceProps) => {
             </td>
           </tr>
         )}
+        {experiment.recipeJson && (
+          <tr>
+            <th>Recipe JSON</th>
+            <td data-testid="experiment-recipe-json">
+              <JSONPretty
+                id="json-pretty"
+                data={experiment.recipeJson}
+              ></JSONPretty>
```

Comment:
Is `JSONPretty` doing anything for us beyond prettifying and optional syntax highlighting? Since we're not using highlighting I think we can do without the extra dependency and just do something like:

```jsx
<code>
  <pre>
    {JSON.stringify(JSON.parse(experiment.recipeJson), null, 2)}
  </pre>
</code>
```

Reply:
Good call 👍

Reply:
If the server can return it prettified we could even forgo the whole `JSON.stringify(JSON.parse(` business

Reply:
I wish bootstrap had a class to highlight json.  It used to.  Oh well.